### PR TITLE
Fix: Collection detail view has a huge top bar

### PR DESCRIPTION
**Pull request by billypchan**
### Issues

Collection image view has a huge top bar.

### Causes

The navigation bar is doubled when `ConversationImagesViewController` is pushed to a navigation controller.

### Solutions

~~Check a new navigation bar when `viewWillAppear` instead of `viewDidLoad`.
Update `FullscreenImageViewController`'s top overlay visibility after `showCloseButton` flag is updated.~~
Remove the manually created navigation bar in `ConversationImagesViewController`. Always wraps `ConversationImagesViewController` in a navigation controller when it is presented.

### Notes

`Cartography` is removed from `ConversationImagesViewController`.

### CODE REVIEW

**Comment by typfel** [comment]
Any reason why would couldn't always present the view controller in a navigation controller?

**Comment by billypchan** [comment]
I think it is a legacy issue. I will check which view presents `ConversationImagesViewController` without wrapping in a navigation controller.

File: Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m [hunk 1]
```diff
@@ -120,6 +120,14 @@ - (instancetype)initWithMessage:(id<ZMConversationMessage>)message
     return self;
 }
 
+-(void)setShowCloseButton:(BOOL)showCloseButton
```
Comment (typfel) [inline Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m]: Is this method still called?

Comment (typfel) [reply Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m]: And if it's not do we still need the `closeButton` and `topOverlay`?

Comment (billypchan) [reply Wire-iOS/Sources/UserInterface/Components/Controllers/FullscreenImageViewController.m]: I am removing them, thanks!

**Pull request merged**